Formula/tektoncd-cli.rb

```diff
@@ -1,9 +1,9 @@
 class TektoncdCli < Formula
   desc "CLI for interacting with TektonCD"
   homepage "https://github.com/tektoncd/cli"
-  url "https://github.com/tektoncd/cli/archive/v0.8.0.tar.gz"
+  url "https://github.com/tektoncd/cli/archive/v0.9.0.tar.gz"
 
```

Comment:
Please could you remove this blank line? You can amend your commit and force-push the branch. Then I think this is good to go!

Reply:
Addressed. Thank you!